vnext/src/IntegrationTests/AsyncStorageTest.windesktop.js

```diff
@@ -138,13 +157,17 @@ function testRemoveItem() {
 
 function testMerge() {
   if (AsyncStorage.mergeItem) {
-    AsyncStorage.setItem(KEY_MERGE, JSON.stringify(VAL_MERGE_1), err1 => {
+    AsyncStorage.setItem(KEY_MERGE, stringify(VAL_MERGE_1), err1 => {
       expectAsyncNoError('testMerge/setItem', err1);
-      AsyncStorage.mergeItem(KEY_MERGE, JSON.stringify(VAL_MERGE_2), err2 => {
+      AsyncStorage.mergeItem(KEY_MERGE, stringify(VAL_MERGE_2), err2 => {
         expectAsyncNoError('testMerge/mergeItem', err2);
         AsyncStorage.getItem(KEY_MERGE, (err3, result) => {
           expectAsyncNoError('testMerge/setItem', err3);
-          expectEqual(JSON.parse(result), VAL_MERGE_EXPECT, 'testMerge');
+          expectEqual(
+            JSON.parse(nullthrows(result)),
```

Comment:
>nullthrows [](start = 23, length = 10)

In this cases the use of nullthrows could be justified because it produces different result from the default behavior unlike in the cases above where we are getting a property from the variable.

Reply:
Forked file.